superset/viz.py

```diff
@@ -1208,14 +1208,21 @@ class DistributionPieViz(NVD3Viz):
     verbose_name = _("Distribution - NVD3 - Pie Chart")
     is_timeseries = False
     fieldsets = ({
-        'label': None,
+        'label': 'Criterion',
         'fields': (
             'metrics', 'groupby',
             'limit',
+        ),
+        "description": _("Criterion setting of default viewport.")
+    }, {
+        'label': 'Chart Options',
+        'fields': (
             'pie_label_type',
             ('donut', 'show_legend'),
-            'labels_outside',
-        )
+            ('labels_outside', 'pie_half_circle'),
+            ('pie_pad_angle', 'pie_corner_radius'),
+        ),
+        "description": _("Properties setting of default viewport.")
```

Comment:
To have your reply, it's my pleasure.

1.The screenshots 
You can find the screenshots in #1684 .

2.I update my code.please have a check again.

3.I do not know how to change the files in exploreV2,and do not know how to test it in V2.
I issue this before #1677.

Look forward to your reply.